runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java

```diff
@@ -108,7 +110,30 @@ private void renderProgressCircle(Graphics2D graphics, XpGlobe skillToDraw, int
 
 		Ellipse2D backgroundCircle = drawEllipse(graphics, x, y);
 
+		drawSkillImage(graphics, skillToDraw, x, y);
+
+		Point mouse = client.getMouseCanvasPosition();
+		int mouseX = mouse.getX() - bounds.x;
+		int mouseY = mouse.getY() - bounds.y;
+
 		Object renderHint = graphics.getRenderingHint(RenderingHints.KEY_STROKE_CONTROL);
+		graphics.setRenderingHint(RenderingHints.KEY_STROKE_CONTROL, renderHint);
```

Comment:
Hmm? What is this for

Reply:
This was here before so he probably would not know

Reply:
`graphics.setRenderingHint(RenderingHints.KEY_STROKE_CONTROL, renderHint);` was definitely not here. It was after VALUE_STROKE_PURE where it makes sense, but here it doesnt